server.ts

```diff
@@ -9,14 +10,23 @@ const dev = process.env.NODE_ENV !== "production";
 const app = next({ dev });
 const handle = app.getRequestHandler();
 
+Sentry.init({
+  dsn: "https://320d89774d7344ed9c949a784a6e1c7d@sentry.io/1325279"
+});
+
 app
   .prepare()
   .then(() => cardsUtils.setupCardsStorage())
   .then(() => {
     const server = express();
+    server.use(Sentry.Handlers.requestHandler() as express.RequestHandler);
 
+    // The error handler must be before any other error middleware
+    server.use(Sentry.Handlers.errorHandler() as express.ErrorRequestHandler);
     server.use(api);
 
+    // The error handler must be before any other error middleware
+    server.use(Sentry.Handlers.errorHandler() as express.ErrorRequestHandler);
```

Comment:
Do we really need to add these two blocks twice?

Reply:
Comment above

Reply:
>     // The error handler must be before any other error middleware

Reply:
https://sentry.io/for/express/